spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -151,6 +154,9 @@
       to receive(:get_usps_proofing_results_job_request_delay_milliseconds).
       and_return(request_delay_ms)
     stub_request_token
+    if defined?(pending_enrollment)
+      pending_enrollment.update(enrollment_established_at: Time.zone.now - 3.days)
+    end
```

Comment:
1. as a habit, I usually use `respond_to?` when checking for method, and `defined?` when checking for constants
2. I think we can use `3.days.ago` instead of subtracting

```suggestion
    if respond_to?(:pending_enrollment)
      pending_enrollment.update(enrollment_established_at: 3.days.ago)
    end
```